test/test_forms.jl

```diff
@@ -48,4 +51,19 @@ using Random
 
   @test underlying_graph(ket_network(qf)) == underlying_graph(ψket)
   @test underlying_graph(operator_network(qf)) == underlying_graph(A)
+
+  ∂qf_∂v = only(derivative_state(qf, [v]))
+  @test (∂qf_∂v) * (qf[ket_vertex_map(qf)(v)] * qf[bra_vertex_map(qf)(v)]) ≈
+    ITensors.contract(qf)
+
+  ∂qf_∂v_bp = derivative_state(qf, [v]; alg="bp", update_bp_cache=false)
+  ∂qf_∂v_bp = ITensors.contract(∂qf_∂v_bp)
```

Comment:
```suggestion
  ∂qf_∂v_bp = contract(∂qf_∂v_bp)
```

Reply:
If the issue is that `contract` conflicts with one defined in `Graphs`, we should instead do `using Graphs: nv, ...` to bring the functions from `Graphs` into the namespace that we are actually using in the tests.

Reply:
Yes this appears to be due to `WARNING: both Graphs and ITensors export "contract"; uses of it in module Main must be qualified` I get on compilation. Let me implement your suggestion.